pkg/gardenlet/apis/config/v1alpha1/defaults.go

```diff
@@ -408,6 +408,11 @@ func SetDefaults_ManagedSeedControllerConfiguration(obj *ManagedSeedControllerCo
 		v := metav1.Duration{Duration: 5 * time.Minute}
 		obj.SyncJitterPeriod = &v
 	}
+
+	if obj.JitterUpdates == nil {
+		falseVar := false
+		obj.JitterUpdates = &falseVar
```

Comment:
Any specific reason you have not used `pointer.Bool(false)`

Reply:
I think both of them are fine.

Reply:
I agree with @acumino (even though it's a nit): `pointer.Bool` is more precise.

Reply:
Addressed in [commit](https://github.com/gardener/gardener/pull/5483/commits/de284d6ecaa02c1411dda2dee6accfbb949d97f5).